panels/models.py

```diff
@@ -89,12 +93,17 @@ def submitter(self):
             return submitter
 
     @property
-    def matching_attendees(self):
-        return [a.matching_attendee for a in self.applicants if a.matching_attendee]
+    def matched_attendees(self):
+        return [a.attendee for a in self.applicants if a.attendee_id]
+
+    @property
+    def unmatched_applicants(self):
+        return [a for a in self.applicants if not a.attendee_id]
 
 
 class PanelApplicant(MagModel):
     app_id = Column(UUID, ForeignKey('panel_application.id', ondelete='cascade'))
+    attendee_id = Column(UUID, ForeignKey('attendee.id', ondelete='cascade'), nullable=True)
```

Comment:
This is setting the stage for later work.  We want to be able to directly associate panel applicants with the attendees which are created for them.  For now we just added the nullable column, and later on we'll do the work to fill it in.